posthog/mixins.py

```diff
@@ -3,6 +3,28 @@
 from posthog.event_usage import report_user_action
 
 
+def log_deletion_metadata_to_posthog(func):
+    """
+    wraps a DRF destroy endpoint and sends a PostHog event recording its deletion
+    so:
+        * args[0] is the ViewSet
```

Comment:
Given that this is what we expect, can we explicitly add to the wrapper signature/typing (and func I guess)?